domains/nikywelcome.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "nikyy2",
+        "email": "anhky581@gmail.com"
+    },
+    "record": {
+        "CNAME": "28nikywelcome03.github.io"
```

Comment:
to link the site you would need to use nikytap1n.github.io instead of this CNAME (since the link starts with the domain im suggesting you to add)